extensions/test-extension/src/test_extension.rs

```diff
@@ -36,9 +49,21 @@ impl TestExtension {
             },
         )?;
 
-        let (platform, arch) = zed::current_platform();
+        let ext;
+        let download_type;
+        match platform {
+            zed::Os::Mac | zed::Os::Linux => {
+                ext = "tar.gz";
+                download_type = zed::DownloadedFileType::GzipTar;
+            }
+            zed::Os::Windows => {
+                ext = "zip";
+                download_type = zed::DownloadedFileType::Zip;
+            }
+        }
+
```

Comment:
```suggestion
        let (ext, download_type) = match platform {
            zed::Os::Mac | zed::Os::Linux => (
                "tar.gz",
                zed::DownloadedFileType::GzipTar,
            )
            zed::Os::Windows => (
                "zip",
                zed::DownloadedFileType::Zip,
            )
        };

```

Reply:
I ended up getting rid of this code, since the integration test is only set up to deal with a `tar.gz`, but left a comment for what to do if you want to actually install this extension to manually test.